nessie/src/main/java/org/apache/iceberg/nessie/NessieCatalog.java

```diff
@@ -70,12 +74,12 @@
  * </p>
  */
 public class NessieCatalog extends BaseMetastoreCatalog implements AutoCloseable, SupportsNamespaces, Configurable {
-  private static final Logger logger = LoggerFactory.getLogger(NessieCatalog.class);
+  private static final Logger LOGGER = LoggerFactory.getLogger(NessieCatalog.class);
```

Comment:
The standard is to name it `LOG`

Reply:
fixed. Perhaps we should add a checkstyle for this? Seems to be enforced sporadically in the codebase atm